libcloud/test/dns/test_nsone.py

```diff
@@ -0,0 +1,259 @@
+import sys
+import unittest
+
+from libcloud.test import MockHttp
+from libcloud.test.file_fixtures import DNSFileFixtures
+from libcloud.test.secrets import DNS_PARAMS_NSONE
+from libcloud.dns.drivers.nsone import NsOneDNSDriver
+from libcloud.utils.py3 import httplib
+from libcloud.dns.types import ZoneDoesNotExistError, ZoneAlreadyExistsError,\
+    RecordDoesNotExistError,  RecordType
```

Comment:
There are 2 spaces after the comma. this is failing the lint. remove one of the spaces and just commit that change then we're good to go @jetbird